### PR TITLE
Accessibility add logos name

### PR DESCRIPTION
**Pull request by Arhell**
resolve https://github.com/fluent/fluentd-website/issues/194

### CODE REVIEW

**Comment by Arhell** [comment]
cc @ashie

File: views/index.erb [hunk 1]
```diff
@@ -82,67 +82,67 @@
       <!--<div class="headline"><h2></h2></div>-->
 		  <ul class="slides">
 			  <li>
-          <a href="testimonials">
+          <a href="testimonials" aria-label="logotype">
             <img src="/images/testimonials/line.png" alt="" />
           </a>
```
Comment (ashie) [inline views/index.erb:87]: I'm not sure but setting `alt` to all images doesn't solve this issue?

Comment (ashie) [inline views/index.erb:87]: I'm not sure but setting `alt` to all images doesn't solve this issue?

Comment (Arhell) [reply views/index.erb:87]: Unfortunately no.

Comment (Arhell) [reply views/index.erb:87]: Unfortunately no.

Comment (Arhell) [reply views/index.erb:87]: @ashie i can add not "logotype" => company

**Comment by Arhell** [comment]
update aria & add alt

**Comment by ashie** [comment]
Thanks!

**Pull request merged**